src/cc/bpf_module.cc

```diff
@@ -95,6 +98,8 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags)
     : flags_(flags), ctx_(new LLVMContext) {
+  InitializeNativeTarget();
```

Comment:
I was hoping we can eventually get rid of x86 backend dependency, but this forces it in. Why do we need it?